### PR TITLE
setup cypress configuration

### PR DESCRIPTION
**Pull request by DanielCalleCO**

### CODE REVIEW

File: cypress/support/e2e.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (danielgalvis98) [inline cypress/support/e2e.js:20]: EOF

File: cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```
Comment (danielgalvis98) [inline cypress/support/commands.js:25]: EOF

File: cypress/fixtures/profile.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (danielgalvis98) [inline cypress/fixtures/profile.json:1]: I think there is something weird, maybe you pressed another button or command? I think this file shouldn't be here, please delete it

File: cypress/fixtures/users.json [hunk 4]
```diff
@@ -0,0 +1,232 @@
+[
```
Comment (danielgalvis98) [inline cypress/fixtures/users.json:1]: Same comment, you can delete this file

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**